src/Task1/SelectGenre.jsx

```diff
@@ -29,44 +34,48 @@ const SelectGenre = () => {
           }}
         >
           Select Genre
-          <span style={{ clipPath: clip }} className="arrow">
-            ok
-          </span>
+          <span className={`arrow ${isopen ? "opened" : "notopen"}`}>ok</span>
         </span>
 
         <ul className="items" style={{ marginTop: "20px" }}>
-          <li>
-            <label className="container">
-              Crime
-              <input type="checkbox" />
-              <span class="checkmark"></span>
-            </label>
-          </li>
-          <li>
-            <label className="container">
-              Documentary
-              <input type="checkbox" />
-              <span class="checkmark"></span>
-            </label>
-          </li>
-          <li>
-            <label className="container">
-              Horror
-              <input type="checkbox" />
-              <span class="checkmark"></span>
-            </label>
-          </li>
-          <li>
-            <label className="container">
-              Comedy
-              <input type="checkbox" />
-              <span class="checkmark"></span>
-            </label>
-          </li>
+          {genre.map((el, i) => {
+            return (
```

Comment:
`el` is obscure name for this variable. You should probably name genres array as `genres` and then use `genre` in `map()` callback instead of `el`.